src/elements/container/sticky-bar/sticky-bar.scss

```diff
@@ -61,6 +61,14 @@ $intersector-overlapping: 1px;
   --sbb-sticky-bar-sticky-background-color: var(--sbb-color-milk);
 }
 
+:host([data-sticking]:not([data-state='unsticky'])[color='midnight']) {
```

Comment:
@mcilurzo Do we need a change for shadow color when sticky? Or does the shadow of the milk and white variant still work? https://lyne-components-pr3337.app.sbb.ch/?path=/story/elements-sbb-container-sbb-sticky-bar--midnight-container

Reply:
in my opinion, the shadow still works.